### DISCUSSION TITLE
Increment a feature from the release

### DISCUSSION THREAD

**Discussion by lbensch** [GitTools/GitVersion#3145] [Q&A] [answered]
Hello,
I have a release/v3.80.0.0 branch
When I create a feature branch from this branch, the version number is based on the last known tag and not on the name of the branch from which it starts.
Is there any way to make my version number 3.80?

**Comment by asbjornu** [comment]
You can try to configure `feature` branches to consider `release` branches as their [`source-branches`](https://gitversion.net/docs/reference/configuration#source-branches).

**Comment by HHobeck** [comment, marked as answer]
It might be a good idea to write an integration test to see exactly what you are expecting. I think that in your scenario the PR #3190 is going to do what you want. I'm right?
